Assets/Scripts/Gamemode/CaptureGamemode.cs

```diff
@@ -103,6 +103,11 @@ public void AddTitanScore(int score)
             CheckWinConditions();
         }
 
+        private void awake()
```

Comment:
Please remove this

Reply:
I think I accidentally applied stashed changes for issue #175 as I was messing around with it. This isnt supposed to be in there